src/helpers/format/format.ts

```diff
@@ -64,13 +66,22 @@ export const DAYS: Readonly<Record<number, string>> = {
   6: _t("Saturday"),
 };
 
+interface FormatWidth {
+  availableWidth: number;
+  measureText: (text: string) => number;
+}
+
 /**
  * Formats a cell value with its format.
  */
-export function formatValue(value: CellValue, { format, locale }: LocaleFormat): FormattedValue {
+export function formatValue(
```

Comment:
can we have at least a few tests for this feature in `format_helpers.test.ts` ? (with mocked measures)